lib/celluloid/groups/spawner.rb

```diff
@@ -19,9 +19,12 @@ def get(&block)
       end
 
       def each
-        threads = []
-        @mutex.synchronize { threads = @group.dup }
-        threads.each { |thread| yield thread }
+        to_a.each { |thread| yield thread }
+      end
+
+      # For temp compatiblity with specs
```

Comment:
Until the specs are "stabily green" at least, and then until I've ported the "fixed" InternalPool - because this API may be unnecessary, but it's good at testing the stuff.

Either way there's probably more refactoring than this - and refactoring is done best when stuff is green - also because while refactoring you might find bugs, so it's best to do it on a "pre-release ready" codebase.